v1.3 RC omnibus (#2104)
fishtown-analytics/docs.getdbt.com: website/docs/guides/migration/versions/05-upgrading-to-v1.3.md

## Breaking changes

There are no breaking changes for code in dbt projects and packages. We are committed to providing backward compatibility for all versions 1.x. If you encounter an error upon upgrading, please let us know by [opening an issue](https://github.com/dbt-labs/dbt-core/issues/new).

### For users of dbt Metrics

The names of metric properties have changed, with backward compatibility. Those changes are:
- Renamed `type` to `calculation_method`
- Renamed `sql` to `expression` 
- Renamed `expression` calculation method metrics to `derived` calculation method metrics

We plan to keep backward compatibility for a full minor version. Defining metrics with the old names will raise an error in dbt Core v1.4.

### For consumers of dbt artifacts (metadata)

> The manifest schema version has been updated to `v7`. This includes the changes to metrics described above, and a few other changes related to the addition of Python models:

Reviewer: suggest active voice

```suggestion
We have updated the manifest schema version to `v7`. This includes the changes to metrics described above and a few other changes related to the addition of Python models:
```
